caf-audit-service-container/pom.xml

```diff
@@ -62,7 +62,7 @@
         </dependency>
         <dependency>
             <groupId>com.github.cafaudit</groupId>
-            <artifactId>caf-audit</artifactId>
+            <artifactId>caf-audit-service-testing-audit</artifactId>
```

Comment:
I have renamed this to "-auditlog" in the latest commit. Changing it to caf-audit-service-testing sounds like it contains tests for the audit service.